cypress/e2e/favorites-spec.cy.js

```diff
@@ -67,7 +67,7 @@ describe('Favorites Page', () => {
     cy.get('[data-cy="fav-user-message"]').contains(`You don't have any favorites yet`);
   })
   
-  it('a user should be able to add a favorite and search through favorites', () => {
+  it.only('a user should be able to add a favorite and search through favorites', () => {
```

Comment:
We should remove the 'only' from line 70 in favorites-spec.cy.js

Reply:
ah thankyou for looking at that. I can work on that today